Project/game.py

```diff
@@ -50,7 +51,8 @@ def print_cli(self, player):
         os.system("clear")
         player_no = self.players.index(player) + 1
         print("Player: %d" % player_no)
-        print("Current Suit: %s Current Card: %s\n" % (self.current_suit, self.current_card))
+        if self.current_card == 0: print("Current Suit: %s" % self.current_suit)
```

Comment:
I'm making this either print the suit or the card so the player isn't confused if the suit is different than the suit of the card. Also, if the suit is changed on after an 8 is laid, the next player has to play the same suit. The number of the card won't matter.